src/Lecture1.hs

```diff
@@ -123,4 +137,17 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: (Show a, Ord a) => a -> [a] -> [Char]
+lowerAndGreater n list =
+        iter n list 0 0
+        where
+            iter n [] lower greater = show n
```

Comment:
I see that the number `n` that you pass in recursive calls doesn't change. You always pass the same number. In that case, you can remove this extra argument and use `n` from the function instead. It's possible because functions in `where` can see all the arguments of the top-level function 🔍